### PR TITLE
Test functionality

### PR DESCRIPTION
**Pull request by binki**
Includes the commit from #11 because, without that, the included snapshots would be wrong.

These tests try to verify that `flow-result-checker` actually accomplishes what it tries to do. The next step is to get these tests running on Windows (my goal is to get it working at all on Windows ;-))…

I’m not sure what the right way to do things is because I’ve made a bunch of PRs which are all sort of relying on each other, hopefully some of them get merged and if some don’t I can rebase the others after fixing with any feedback.

### CODE REVIEW

**Comment by jbreckel** [comment]
this looks good. I merged a request for the `No Errors` case.

Please rebase yours and see if it works now. 🙂

**Comment by binki** [comment]
@jbreckel Hi, I have:

1. Removed the “No Error” commit through rebasing as requested.
2. Finished getting Windows support and fixed ApVeyor CI support.

May you please merge this and, once done, configure appveyor on this repository to run for PRs? I don’t have the right access to turn appveyor on. To do so, simply visit https://ci.appveyor.com/projects , login with GitHub, and then in the project list add flow-result-checker.

After all of this, a verbump and publish should be all that’s needed to get the npm package running on Windows.

P.S.,

I see I was given push access to this repository. Not sure I’ll ever use it xD. Are there any guidelines on what I’m supposed to do with these privileges? I.e., should I still make PRs for most things…? I’ll try my best not to abuse this, thanks! ;-)

**Review by jbreckel — APPROVED** [review]

**Pull request merged**
